### ISSUE TITLE
As PO I want to migrate the API Testplatform from Google Cloud to Haven

### ISSUE THREAD

**Issue by HenriKorver** [VNG-Realisatie/api-testvoorziening#170]
such that VNG has full control over the hosting environment.

**Toelichting**:
...

**Oplossingsrichting**:
Zie https://github.com/VNG-Realisatie/gemma-zaken/issues/973

**Definition of ready**
- [ ] Iedereen in het team begrijpt de user story
- [ ] Is klein genoeg (maximaal 1/3 van sprint)
- [ ] Product Owner akkoord en voorzien van prioriteit (mag alleen afgevinkt worden door Henri)
- [ ] Voorzien van Acceptatiecriteria (duidelijk en testbaar)
- [ ] Voorzien van Definition of Done (duidelijk en testbaar)
- [ ] Voorzien van taken
- [ ] Idee hebben van hoe deze user story kan worden gedemonstreerd.
- [ ] Vastgelegd in Github en geplaatst in kolom ready

**Definition of done algemeen**
- [ ] Tekst Aanbesteding

**Definition of done specifiek voor deze user story**
- [ ] ....

**Acceptatiecriteria**
- [ ] ...
- [ ] ...
- [ ] ...

**Taken**
Taken worden apart uitgewerkt in issues.

**Comment by alextreme** [comment]
Momenteel maken we gebruik van (1) een dedicated server van Maykin voor het hosten van het testplatform, en (2) maken we gebruik van Google Cloud voor het automatisch starten van Docker pods voor testsessies.

Welke van de 2 bedoel je? Zoals aangegeven is het automatisch opstarten van de Docker pods voor testsessies sterk gelinkt aan Google Cloud (wordt via 'gcloud' uitgevoerd). Ik ga er dan ook vooralsnog vanuit dat je met deze story (1) bedoelt, ook al draait het ATV zelf niet op Google Cloud.

**Comment by HenriKorver** [comment]
@alextreme Bedankt voor de verscherping. Ik bedoelde eigenlijk beide, dus zowel 1 als 2. Zie jij complicaties om ook 2) naar Haven te verhuizen?

**Comment by alextreme** [comment]
Ja, (2) zal een stuk meer tijd vergen aangezien het betekent dat we de huidige GCP commando's in het testplatform moeten vervangen voor Haven. 

Als je (via Roderick?) Alvise en mij toegang kan geven dan kunnen we de impact bepalen van beide onderdelen. Ik zou wel aanraden om deze story te slicen.

**Comment by HenriKorver** [comment]
@kciredor Kun jij @AlviseSembenico en @alextreme toegang geven tot Haven om impact te bepalen?

**Comment by HenriKorver** [comment]
@kciredor kan nog geen toegang geven omdat Have nog niet af is (binnen enkele weken wel). Maar op slack geeft hij allerlei tips om alvast voorbereidingen te treffen om zo zacht mogelijk op Haven te kunnen landen.
(Je kunt bijvoorbeeld minikube zelfs gebruiken om Kubernetes compatible te ontwikkelen aan APITV, als het in minikube werkt, werkt het ook in Haven.)

**Comment by HenriKorver** [comment]
>  Ik zou wel aanraden om deze story te slicen.

Done (op gesplitst in deze US en #171)
